### PR TITLE
feat(eks): add helm flag --skip-crds

### LINKED ISSUE #24296: (eks): (helm flag --skip-crds)
### Describe the feature

Helm provides the feature flag to skip the installation of custom resource definitions.

This flag could be added to the helm chart construct.

### Use Case

When installing Helm chart sometimes it is needed to skip the installation of custom resource definitions. (e.g. when you are managing those crds by yourself)

### Proposed Solution

Adding the flag `--skip-crds` to the Helmchart Construct could 

### Other Information

_No response_

### Acknowledgements

- [X] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

2

### Environment details (OS name and version, etc.)

Mac OS 13.2

### PR DESCRIPTION
**Pull request by alexandersperling**
When installing Helm chart sometimes it is needed to skip the installation of custom resource definitions.
This MR adds the functionality to the helm chart construct.

In fact I did the following

* Add the possibility to skip the installation of crds in a helm chart via a new property
* Add tests for skipCrds property
* Update README

Closes #24296 

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by TheRealAmazonKendra** [comment]
Thank you for your contribution. I'll get these run today.

**Comment by TheRealAmazonKendra** [comment]
Apologies for the delay on this. I was helping troubleshoot another EKS issue that I suspected might impact this one. I didn't, but it took a while to get it resolved. Running those tests now!

File: packages/@aws-cdk/aws-eks/README.md [hunk 1]
```diff
@@ -1193,6 +1193,20 @@ cluster.addHelmChart('ExternalSecretsOperator', {
...
+
+```ts
+declare const cluster: eks.Cluster;
+// option 1: use a construct
+new eks.HelmChart(this, 'NginxIngress', {
+  cluster,
+  chart: 'nginx-ingress',
+  repository: 'https://helm.nginx.com/stable',
+  namespace: 'kube-system',
+  skipCrds: true,
```
Comment (TheRealAmazonKendra) [inline packages/@aws-cdk/aws-eks/README.md:1206]: Is CRDs a typical convention for this? If not, I'd prefer to have this spelled out so it's more clear.

**Comment by TheRealAmazonKendra** [comment]
@Mergifyio update

File: packages/@aws-cdk/aws-eks/README.md (hunk 1 above)
Comment (alexandersperling) [reply packages/@aws-cdk/aws-eks/README.md:1206]: Good point, I checked the docs of k8s and helm and it seems that this is a common convention.

[k8s custom resource definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
[helm custom resource definition](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/)

**Comment by TheRealAmazonKendra** [comment]
@Mergifyio update

File: packages/@aws-cdk/aws-eks/README.md (hunk 1 above)
Comment (TheRealAmazonKendra) [reply packages/@aws-cdk/aws-eks/README.md:1206]: In that case, I consider this good to go!

**Review by TheRealAmazonKendra — APPROVED** [review]

**Pull request merged**